src/frontend/citizen-portal/src/app/services/violation-ticket.service.ts

```diff
@@ -38,6 +37,7 @@ export class ViolationTicketService {
     "counts.count_no_3.is_regulation", "counts.count_no_3.section", "counts.count_no_3.ticketed_amount", "court_location", "detachment_location"];
   DetectedOcrIssues = DisputeDisputantDetectedOcrIssues;
   SystemDetectedOcrIssues = DisputeSystemDetectedOcrIssues;
+  private _months: string[] = ["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"];
```

Comment:
Please don't declare some variables that were built-in

Reply:
Good feedback. Could you elaborate on the `built-in`. This prevents the author from trying to figure it out.

Reply:
it does not